test/core-flags/optimization-flags.test.js

```diff
@@ -21,20 +25,24 @@ describe('optimization config related flag', () => {
                 if (flag.name === 'optimization-split-chunks') {
                     expect(stdout).toContain(`chunks: 'async'`);
                     expect(stdout).toContain(`minChunks: 1`);
+                } else if (flag.name.includes('reset')) {
+                    expect(stdout).toContain(`${propName}: []`);
                 } else {
                     expect(stdout).toContain(`${propName}: true`);
                 }
             });
 
-            it(`should config --no-${flag.name} correctly`, () => {
-                const { stderr, stdout } = run(__dirname, [`--no-${flag.name}`]);
-                expect(stderr).toBeFalsy();
-                if (flag.name === 'optimization-split-chunks') {
-                    expect(stdout).toContain('splitChunks: false');
-                } else {
-                    expect(stdout).toContain(`${propName}: false`);
-                }
-            });
+            if (!flag.name.includes('reset')) {
+                it(`should config --no-${flag.name} correctly`, () => {
+                    const { stderr, stdout } = run(__dirname, [`--no-${flag.name}`]);
```

Comment:
Right now we support negative flags for every boolean flag including the `reset` flags, for example `--no-externals-reset` is valid but it doesn't do anything. I think we should not support negative reset flags. 
If no one objects, I can create a separate PR for that.